pyanaconda/core/configuration/anaconda.py

```diff
@@ -105,6 +105,111 @@ def _is_unknown(self):
         """Are we running in the unknown OS?"""
         return self._type is SystemType.UNKNOWN
 
+    @property
+    def can_reboot(self):
+        """Can we reboot the system?"""
+        return self._is_boot_iso
+
+    @property
+    def can_switch_tty(self):
+        """Can we change the foreground virtual terminal?"""
+        return self._is_boot_iso
+
+    @property
+    def can_audit(self):
+        """Can we run the audit daemon?"""
+        return self._is_boot_iso
+
+    @property
+    def can_adjust_time(self):
+        """Can we change the time?"""
+        return self._is_boot_iso
+
+    @property
+    def can_adjust_time_live(self):
+        """Can we change the time?
+
+        FIXME: Conflict with can_adjust_time.
+        """
+        return self._is_boot_iso or self._is_live_os
+
+    @property
+    def can_synchronize_time(self):
+        """Can we run the NTP daemon?"""
+        return self._is_boot_iso
+
+    @property
+    def can_localize(self):
+        """Can we change the localization?"""
+        return self._is_boot_iso
+
+    @property
+    def can_localize_live(self):
+        """Can we change the localization?
+
+        FIXME: Conflict with can_localize.
+        """
+        return self._is_boot_iso or self._is_live_os
+
+    @property
+    def can_configure_syslog(self):
+        """Can we modify syslog?
+
+        FIXME: This rule is weird.
+        """
+        return self._is_boot_iso
+
+    @property
+    def can_modify_nvram(self):
+        """Can we modify firmware NVRAM variables?
+
+        FIXME: Isn't this target specific?
+        """
+        return self._is_boot_iso or self._is_live_os
+
+    @property
+    def can_copy_resolve_conf(self):
+        """Can we copy /etc/resolv.conf to the target system?
+
+        FIXME: Isn't this rule too specific?
+        """
+        return self._is_boot_iso
+
+    @property
+    def can_change_hostname(self):
+        """Can we change the hostname?"""
+        return self._is_boot_iso
+
+    @property
+    def can_change_hostname_live(self):
```

Comment:
The same here, there should be just `can_change_hostname` and test for live if needed.

Also again I would like to see here convention with start `can_configure_...`.

Reply:
Replaced with `can_change_hostname` after a discussion with @rvykydal.